spec/timezone.html

```diff
@@ -5,6 +5,65 @@
   <h1>Temporal.TimeZone Objects</h1>
   <p>A Temporal.TimeZone object is an immutable Object referencing a time zone.</p>
 
+  <emu-clause id="sec-time-zone-names">
+    <h1>Time Zone Names</h1>
+
+    <p>
+      An ECMAScript implementation must support a number of built-in time zones.
+      Implementations must support a built-in time zone named *"UTC"* as well as the host environment's current time zone (if different from UTC), and may support any number of other built-in time zones.
+    </p>
+    <p>
+      Implementations are encouraged to support the Zone and Link names of the IANA Time Zone Database, and to use the best available current and historical information about their offsets from UTC and their daylight saving time rules in calculations.
+    </p>
+    <p>
+      An ECMAScript implementation that includes the ECMA-402 Internationalization API has additional requirements for the built-in time zones that must be supported, as specified in the ECMA-402 specification.
+    </p>
+
+    <emu-clause id="sec-isvalidtimezonename" aoid="IsValidTimeZoneName">
+      <h1>IsValidTimeZoneName ( _timeZone_ )</h1>
+
+      <p>
+        An ECMAScript implementation that includes the ECMA-402 Internationalization API must implement the *IsValidTimeZoneName* abstract operation as specified in the ECMA-402 specification.
+        If an ECMAScript implementation does not include the ECMA-402 API the following specification of the *IsValidTimeZoneName* abstract operation is used.
+      </p>
+      <p>
+        The IsValidTimeZoneName abstract operation verifies that the _timeZone_ argument (which must be a String value) represents a valid built-in time zone.
+      </p>
+      <p>
+        The abstract operation returns true if _timeZone_, converted to upper case as described in <emu-xref href="#sec-string.prototype.touppercase"></emu-xref>, is equal to one of the built-in time zone names, converted to upper case as described in <emu-xref href="#sec-string.prototype.touppercase"></emu-xref>. It returns false otherwise.
```

Comment:
What would be the effect of "being one of the built-in time zone names"? In other words, what other things are mandated when this operation returns `true` or `false`, and what other operations' return values mandate a return value of `true` or `false` here?

Also, this seems to overlap heavily with CanonicalizeTimeZoneName - should one of them call the other?